.github/ISSUE_TEMPLATE/bug-performance-report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug/Performance Issue
+about: Use this template for reporting a bug or a performance issue.
+---
+
+**System information**
+- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
+- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
+- TensorFlow installed from (source or binary):
+- TensorFlow version (use command below):
+- TensorFlow Addons installed from (source, PyPi):
+- TensorFlow Addons version:
+- Python version and type(eg. Anaconda Python, Stock Python as in Mac, or homebrew installed Python etc):
+- Bazel version (if compiling from source):
+- GCC/Compiler version (if compiling from source):
+- Is GPU used? : [yes/no]
+- CUDA/cuDNN version (if compiling/running with GPGPU):
+- GPGPU model and memory:
```

Comment:
Can we just call it a GPU? That's how it'll be referred to by most.

Reply:
Also is it necessary to request graphics memory size?

Reply:
Changed the name to just GPU, i think memory would be required

Reply:
Could you elaborate on why memory is required? If we know the model we can easily look up the memory. Also, how helpful to us is that? If there is an OOM crash it'll be apparent in the logs

Reply:
Yes it would save our looking up a bit but no other useful thing i could remove it

Reply:
removed it